src/semgrep_agent/meta.py

```diff
@@ -37,7 +37,7 @@ class GitMeta:
     @cachedproperty
     def event_name(self) -> str:
         if self.pr_id:
-            return "pull-request"
+            return "pull_request"
```

Comment:
fixes a bug from before where github PRs were "on": "pull_request" and non-github were "on": "pull-request"